SonarRunner.Shim/RoslynV1SarifFixer.cs

```diff
@@ -30,7 +39,7 @@ public static class RoslynV1SarifFixer
         /// <summary>
         /// Returns true if the input is parseable JSON. No checks are made for conformation to the SARIF specification.
         /// </summary>
-        public /* for test */ static bool IsValidJson(string input)
+        private bool IsValidJson(string input)
         {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The 'this' parameter (or 'Me' in Visual Basic) of 'RoslynV1SarifFixer.IsValidJson(string)' is never used. Mark the member as static (or Shared in Visual Basic) or use 'this'/'Me' in the method body or at least one property accessor, if appropriate. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3AMarkMembersAsStatic)